Formula/cfitsio.rb

```diff
@@ -39,4 +40,18 @@ def install
       end
     end
   end
+
+  test do
+    cp Dir["#{pkgshare}/testprog/testprog*"], testpath
+    flags = (ENV.cflags || "").split + (ENV.cppflags || "").split + (ENV.ldflags || "").split
```

Comment:
Mike - do you know why this propagation is necessary in the first place? It seems like this is sort of bypassing environment sanitization. Should we even be picking up user-defined compilation flags when doing a test of a build tool package? Or is this doing something else related to how `brew test` sets up the environment?